reg/drone/service/actuator/esc/_.0.1.uavcan

```diff
@@ -0,0 +1,176 @@
+#      This control mode emulates the behavior of a typical RCPWM-controlled BLDC drive.
+#
+#   1. Ratiometric current/torque control. Each setpoint scalar is a value normalized/saturated in [-1, +1] representing
+#      the Q-axis/phase/armature (depending on the type of the drive) current as a fraction of the maximum.
+#      A negative setpoint during forward rotation (positive during reverse rotation) commands braking.
+#
+#   2. Speed control. Each setpoint scalar contains the target angular velocity of the load in radian/second.
+#
+#   -. More control modes may be added later. Which control modes are supported is implementation-defined.
+#
+# Considerations that apply to all control modes:
+#  -  Negative setpoint values represent reversal; a positive setpoint is co-directed with positive rotation/torque.
+#  -  If reverse operation is not supported, negative values should be clamped to zero.
+#  -  A non-finite setpoint is to be treated as zero.
+#
+#
+#   ARMING SUBJECT
+#
+# The default state is DISARMED. While disarmed, the drive is not allowed to deliver power to the load,
+# and the setpoint subject is ignored. The drive shall enter the disarmed state automatically if the arming subject
+# is not updated for CONTROL_TIMEOUT.
+#
+# While the drive is ARMED, the setpoint commands are processed normally as described in the adjacent section.
+# If the drive does not support bidirectional operation, implementations are recommended to ensure that the load
+# is driven at some minimum power level (idling) while the drive is armed regardless of the commanded setpoint,
+# unless such behavior is deemed incompatible with the functional requirements of the controlled drive.
+#
+# If the selected arming state is SLEEP, the behavior is implementation-defined. Implementations are recommended to
+# power off the high-voltage circuitry and all non-essential components (e.g., LED indication, sensors, etc.)
+# to minimize the power consumption.
+#
+# Implementations are recommended to announce transitions between the arming states using audiovisual feedback.
+#
+# The worst-case state transition latency is not defined. The controlling element (that is, the unit that publishes
+# to the setpoint and arming subjects) is expected to monitor the actual arming status of each component using
+# the feedback subject. For example, a sensorless electric motor drive may choose to spool-up before entering the
+# ARMED state, which would obviously take time; as soon as the spool-up is finished, the drive would switch its
+# reported status from DISARMED to ARMED, thereby indicating that it is ready for normal operation.
+#
+#
+#   PUBLISHED SUBJECTS
+#
+# The following subjects shall be published immediately after a new setpoint is applied even if the drive is DISARMED:
+#
+#   SUBJECT             RECOMMENDED PRIORITY
+#   ---------------------------------------------
+#   feedback            same as the setpoint
+#   power               second to the setpoint
```

Comment:
Due to reactive power the current can exceed the ESC limits without beeing noticed if only power is tracked. I would prefer to have voltage and current separately.